src/js/button.js

```diff
@@ -1,3 +1,8 @@
+import Component from './component';
+import * as Lib from './lib';
```

Comment:
Yeah, we don't necessarily have to do this change now.
I'm 100% for breaking up the libs file into separate files.
I believe that you can export each function as needed. Something like:
```js
module 'lib' {
  export let foo = function() {}
  export let bar = function() {}
}

module 'main' {
  import {foo, bar} from 'lib'
}
```
(The `module` syntax is kind of how you can have multiple modules in a single file, but we don't need to worry about that. It's just for demonstration purposes.)